### PR TITLE
[RPA-595] auto version increment - runbotics cli

### PR DESCRIPTION
**Pull request by preludium**

### CODE REVIEW

File: runbotics/runbotics-actions-windows/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "./dist/PowerPointAutomation": "./dist/PowerPointAutomation.js"
     },
     "scripts": {
+        "postinstall": "rushx build",
```
Comment (preludium) [inline runbotics/runbotics-actions-windows/package.json:21]: after a single install we have the whole development environment ready

File: runbotics/runbotics-cli/src/commands/version/version.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import alterFiles from './alter-files';
+import versionControl from './version-control';
+
+const { prompt } = inquirer;
+
+const getBumpType = (versionOptions: VersionOptions) => {
+    const providedOptions = Object.entries(versionOptions).filter(([_, value]) => value === true);
+
+    if (providedOptions.length > 1) {
+        console.log(chalk.red(`${Emoji.error} Error: Options [${providedOptions.map(([key, _]) => `--${key}`).join(', ')}] cannot be used together`));
```
Comment (FranekMarciniak) [inline runbotics/runbotics-cli/src/commands/version/version.ts]: is the _ nesesery?

File: runbotics/runbotics-cli/src/commands/version/version.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        process.exit(1);
+    }
+
+    if (providedOptions.length === 0) return PRERELEASE;
+
+    const [selectedOption, _] = providedOptions[0];
```
Comment (FranekMarciniak) [inline runbotics/runbotics-cli/src/commands/version/version.ts]: same here

File: runbotics/runbotics-cli/src/commands/version/version.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    if (!localConfig.version) {
+        console.log(chalk.red(`${Emoji.error} Error: RunBotics config file is invalid`));
+        process.exit(1);
+    }
+
+    const localVersion = localConfig.version;
+
+    if (!check) {
+        console.log(`${Emoji.skip} Skipping remote version sync`);
+        return [ localVersion, rbRootDir ];
```
Comment (FranekMarciniak) [inline runbotics/runbotics-cli/src/commands/version/version.ts]: Sometimes u use spaces when destructuring array, sometime you dont, pick one

Comment (FranekMarciniak) [reply runbotics/runbotics-cli/src/commands/version/version.ts]: eg line 25

File: runbotics/rush.json [hunk 5]
```diff
@@ -359,28 +359,31 @@
...
+            "packageName": "runbotics-desktop",
+            "projectFolder": "runbotics-desktop",
+            "shouldPublish": true
         },
         {
-            "packageName": "runbotics-sdk",
-            "projectFolder": "runbotics-sdk"
+            "packageName": "runbotics-orchestrator-ui",
+            "projectFolder": "runbotics-orchestrator-ui",
+            "shouldPublish": true
```
Comment (FranekMarciniak) [inline runbotics/rush.json:377]: are u sure about `shouldPublish`?

Comment (FranekMarciniak) [reply runbotics/rush.json:377]: Refering to docs: 
    //    * A flag indicating that changes to this project will be published to npm, which affects
    //    * the Rush change and publish workflows. The default value is false.
    //    * NOTE: "versionPolicyName" and "shouldPublish" are alternatives; you cannot specify them both.

**Review by FranekMarciniak — DISMISSED** [review]

File: runbotics/runbotics-cli/src/commands/version/version.ts (hunk 4 above)
Comment (preludium) [reply runbotics/runbotics-cli/src/commands/version/version.ts]: lack of eslint in the project :D

File: runbotics/rush.json (hunk 5 above)
Comment (preludium) [reply runbotics/rush.json:377]: yep, its not only connected to npm publish but affects commands like `rush change` and `rush version`

File: runbotics/runbotics-cli/src/commands/version/version.ts (hunk 4 above)
Comment (preludium) [reply runbotics/runbotics-cli/src/commands/version/version.ts]: now its there

File: runbotics/runbotics-cli/src/commands/version/local-config.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        });
+
+    const rbConfigPath = join(rbRootDir, RUNBOTICS_CONFIG_RELATIVE_PATH);
+
+    try {
+        const localConfig = readJsonSync(rbConfigPath);
+        return { localConfig, rbRootDir };
+    } catch (e: any) {
+        const message = e.code === 'ENOENT' ? ` (No such file like "${rbConfigPath}")` : undefined;
+        throw new Error(`${Emoji.error} Error: Could not find global config file${message ?? ''}`);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-cli/src/commands/version/local-config.ts:30]: Space between file and message

**Review by FranekMarciniak — APPROVED** [review]

File: runbotics/runbotics-cli/src/commands/version/local-config.ts (hunk 6 above)
Comment (preludium) [reply runbotics/runbotics-cli/src/commands/version/local-config.ts:30]: its intentional, space is included in the message. If no message then no trailing space

**Pull request merged**